src/main/java/org/truffleruby/extra/TruffleGraalNodes.java

```diff
@@ -62,4 +73,49 @@ public DynamicObject splitUnboundMethod(DynamicObject rubyMethod) {
 
     }
 
+    // Like Smalltalk's fixTemps but not mutating the Proc
```

Comment:
@pushrax What do you mean? It's already a comment in the source code.
This method is internal, i.e., should not be used for application code, same for everything under `Truffle`.

Reply:
When reading the internals of Truffle, the context in https://github.com/oracle/truffleruby/pull/271#issuecomment-315451691 is useful for understanding the reason for and limitations of this method. This information doesn't appear to be in any source comment.

Reply:
I see, makes sense, I'll add that information in the source code. I couldn't follow your link the first time, because GitHub only highlights the comment when opening in a new page.

BTW, if you have more questions, you are welcome to ask them in our [Gitter channel](https://gitter.im/graalvm/truffleruby).

Reply:
@pushrax Done in 024f27d04425d8689cb5d2095b1b1a84583662e8